main.cpp

```diff
@@ -1,84 +1,52 @@
-
-    const char* MajorColorNames[] = {
-        "White", "Red", "Black", "Yellow", "Violet"
-    };
-    int numberOfMajorColors =
-        sizeof(MajorColorNames) / sizeof(MajorColorNames[0]);
-    const char* MinorColorNames[] = {
-        "Blue", "Orange", "Green", "Brown", "Slate"
-    };
-    int numberOfMinorColors =
-        sizeof(MinorColorNames) / sizeof(MinorColorNames[0]);
-
-    class ColorPair {
-        private:
-            MajorColor majorColor;
-            MinorColor minorColor;
-        public:
-            ColorPair(MajorColor major, MinorColor minor):
-                majorColor(major), minorColor(minor)
-            {}
-            MajorColor getMajor() {
-                return majorColor;
-            }
-            MinorColor getMinor() {
-                return minorColor;
-            }
-            std::string ToString() {
-                std::string colorPairStr = MajorColorNames[majorColor];
-                colorPairStr += " ";
-                colorPairStr += MinorColorNames[minorColor];
-                return colorPairStr;
-            }
-    };
-
-    ColorPair GetColorFromPairNumber(int pairNumber) {
-        int zeroBasedPairNumber = pairNumber - 1;
-        MajorColor majorColor = 
-            (MajorColor)(zeroBasedPairNumber / numberOfMinorColors);
-        MinorColor minorColor =
-            (MinorColor)(zeroBasedPairNumber % numberOfMinorColors);
-        return ColorPair(majorColor, minorColor);
-    }
-    int GetPairNumberFromColor(MajorColor major, MinorColor minor) {
-        return major * numberOfMinorColors + minor + 1;
-    }
+	TelCoColorCoder::CColorPair colorPair =
+		TelCoColorCoder::GetColorFromPairNumber(pairNumber);
+	std::cout << "Got pair" << colorPair.ColorPairToString() << std::endl;
+	assert(colorPair.GetMajor() == expectedMajor);
+	assert(colorPair.GetMinor() == expectedMinor);
 }
 
-void testNumberToPair(int pairNumber,
-    TelCoColorCoder::MajorColor expectedMajor,
-    TelCoColorCoder::MinorColor expectedMinor)
+/**
+ *  Func Description: testcase to verify match in major and minor color
+ *  with obtained code.
+ */
+void TestPairToNumber(
+	TelCoColorCoder::MajorColor major,
+	TelCoColorCoder::MinorColor minor,
+	int expectedPairNumber)
 {
-    TelCoColorCoder::ColorPair colorPair =
-        TelCoColorCoder::GetColorFromPairNumber(pairNumber);
-    std::cout << "Got pair " << colorPair.ToString() << std::endl;
-    assert(colorPair.getMajor() == expectedMajor);
-    assert(colorPair.getMinor() == expectedMinor);
+	int pairNumber = TelCoColorCoder::GetPairNumberFromColor(major, minor);
+	std::cout << "Got pair number " << pairNumber << std::endl;
+	assert(pairNumber == expectedPairNumber);
 }
 
-void testPairToNumber(
-    TelCoColorCoder::MajorColor major,
-    TelCoColorCoder::MinorColor minor,
-    int expectedPairNumber)
+/**
+ *  Func Description: Test api to get the color code form
+ */
+void TestRefferenceManual() 
```

Comment:
`TestRefferenceManual` is only calling `PrintColorCodesForm`, but it isn't testing the outcome. Any ideas to check the outcome?